src/Angular.js

```diff
@@ -125,6 +128,49 @@ var VALIDITY_STATE_PROPERTY = 'validity';
 
 var hasOwnProperty = Object.prototype.hasOwnProperty;
 
+var objectMaxDepthInErrorMessage = 5;
+
+ /**
+ * @ngdoc function
+ * @name angular.errorHandlingConfig
+ * @module ng
+ * @kind function
+ *
+ * @description
+ * Configure several aspects of error handling in AngularJS if used as a setter or return the
+ * current configuration if used as a getter. The following options are supported:
+ *
+ * - **objectMaxDepth**: The maximum depth to which objects are traversed when stringified for error messages.
+ *
+ * Omitted or undefined options will leave the corresponding configuration values unchanged.
+ *
+ * @param {Object=} config - The configuration object. May only contain the options that need to be
+ *     updated. Supported keys:
+ * -     `objectMaxDepth`  **{Number}** - The max depth for stringifying objects. Setting to a
```

Comment:
When the dash is at the beginning of the line it will be converted to dot

<img width="1280" alt="screen shot 2017-02-03 at 12 32 39 am copy" src="https://cloud.githubusercontent.com/assets/6197215/22571117/0ce5ccae-e9a7-11e6-8363-fe66b3646777.png">

which will be consistent with angular.bootsrtrap config docs

<img width="1280" alt="screen shot 2017-02-03 at 12 40 53 am copy" src="https://cloud.githubusercontent.com/assets/6197215/22571325/fdb3d4be-e9a7-11e6-8a38-4cbd515df66e.png">


But when it is not at the beginning

<img width="1258" alt="screen shot 2017-02-03 at 12 37 37 am copy" src="https://cloud.githubusercontent.com/assets/6197215/22571176/5a946a5a-e9a7-11e6-89e5-fb2553d0b800.png">

Do you want to change it ?